src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -28,6 +28,7 @@ case class WorkspaceName(
                       @(ApiModelProperty@field)(required = true, value = "The name of the workspace")
                       name: String) extends Identifiable {
   def path : String = "workspaces/" + namespace + "/" + name
+  def bucketName : String = Seq("cloud_pilot", namespace, name) mkString "-"
```

Comment:
This is being addressed.  You may want to hold up your review for a few 
minutes until I get the newest code checked in.

On 7/7/15 10:43 AM, dvoet wrote:
>
> In 
> src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala <https://github.com/broadinstitute/rawls/pull/48#discussion_r34047315>:
>
> > @@ -28,6 +28,7 @@ case class WorkspaceName(
> >                        @(ApiModelProperty@field)(required = true, value = "The name of the workspace")
> >                        name: String) extends Identifiable {
> >    def path : String = "workspaces/" + namespace + "/" + name
> > +  def bucketName : String = Seq("cloud_pilot", namespace, name) mkString "-"
>
> I would not use cloud_pilot as a prefix, maybe FireCloud if you want 
> to use something. But I would also append a GUID and store the bucket 
> name as a property in the database.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/broadinstitute/rawls/pull/48/files#r34047315>.
>